documentation/modules/creating-vddk-image.adoc

```diff
@@ -6,16 +6,17 @@
 [id="creating-vddk-image_{context}"]
 = Creating a VDDK image
 
+
 It is strongly recommended that {project-first} should be used with the VMware Virtual Disk Development Kit (VDDK) SDK when transferring virtual disks from VMware vSphere.
```

Comment:
Can we take this line, omve it into the note, and replacing the current text (`Creating a VDDK …`)

Reply:
Instead, I suggest removing the note and leaving "It is strongly recommended..." as is. Please take a look and see what you think.